internal/controllers/utils/utils.go

```diff
@@ -1243,37 +1243,37 @@ func ExtractSubSchema(mainSchema []byte, subSchemaKey string) (subSchema map[str
 		return subSchema, fmt.Errorf("failed to UnMarshall Main Schema: %w", err)
 	}
 	if _, ok := jsonObject[PropertiesString]; !ok {
-		return subSchema, fmt.Errorf("non compliant Main Schema, missing properties: %w", err)
+		return subSchema, fmt.Errorf(`non compliant Main Schema, missing "properties" section: %w`, err)
 	}
 	properties, ok := jsonObject[PropertiesString].(map[string]any)
 	if !ok {
-		return subSchema, fmt.Errorf("could not cast properties as map[string]any: %w", err)
+		return subSchema, fmt.Errorf(`could not cast "properties" section of schema as map[string]any: %w`, err)
```

Comment:
Usually quoting of words in logs is done with a single quote (`'`).

Reply:
Fixed